target/linux/ramips/dts/rt3883_belkin_f9k110x.dtsi

```diff
@@ -102,7 +102,7 @@
 &pci1 {
 	status = "okay";
 
-	wifi@1,0 {
+	wifi@0,0 {
```

Comment:
given that the `reg` value is 0x10000, shouldn't it be wifi@1000,0?

Reply:
same as above.